packages/react-native-ui/src/StorybookLogo.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import { StorybookLogo } from './StorybookLogo';
+import { useTheme } from '@storybook/react-native-theming';
+
+const meta = {
+  component: StorybookLogo,
+  title: 'UI/StorybookLogo',
+  args: {
+    theme: null,
+  },
+} satisfies Meta<typeof StorybookLogo>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const TitleLogo: Story = {
+  decorators: [
+    (Story) => {
+      const theme = useTheme();
+      return <Story args={{ theme: { ...theme, brand: { title: 'React Native' } } }} />;
```

Comment:
I don't like having this decorator in every story just to make typescript happy. Essentially the problem is only that theme requires a lot of properties that we don't want to define here. 
Maybe I missed something to make it easier?

Reply:
https://github.com/trajano/expo-experiments/blob/4287571d3a9fe568700ce97ecc266d5a5bff621e/packages/my-app/.storybook/index.ts#L11-L16 shows what I did on mine to pass it in, but I cheated with the `any` because of the `imageSource` thing which you resolved in web-theme.ts I think.

Reply:
But I think what you have looks correct.  At least for the context of the story.